exporter.go

```diff
@@ -287,24 +303,27 @@ type StatsDListener struct {
 	conn *net.UDPConn
 }
 
-func buildEvent(statType, metric string, value float64, labels map[string]string) (Event, error) {
+func buildEvent(statType, metric string, value float64, valueStr string, labels map[string]string) (Event, error) {
 	switch statType {
 	case "c":
 		return &CounterEvent{
 			metricName: metric,
 			value:      float64(value),
+			valueStr:   valueStr,
```

Comment:
Do we need to store the `valueStr` in all the metric types, or would it be sufficient to just add a `relative` boolean flag only to the `GaugeEvent` type and then just do this:

```go
if event.relative {
    gauge.Add(event.Value())
} else {
  gauge.Set(event.Value())
}
```